src/lib/__tests__/InstantSearch-test.js

```diff
@@ -182,6 +185,11 @@ describe('InstantSearch lifecycle', () => {
     expect(algoliasearch).toHaveBeenCalledWith(appId, apiKey);
   });
 
+  it('calls addAlgoliaAgent', () => {
+    expect(client.addAlgoliaAgent).toHaveBeenCalledTimes(1);
+    expect(client.addAlgoliaAgent).toHaveBeenCalledWith(`instantsearch.js (${version})`);
```

Comment:
This line also needs to be formatted. Running `yarn lint --fix` should fix this kind of problem.

If you use Vscode, you can toggle [format on save](https://marketplace.visualstudio.com/items?itemName=cbrevik.toggle-format-on-save).